RJL.HW4.OOP.Classes.Task2/Workers.cs

```diff
@@ -35,21 +36,22 @@ public int  GetAgregatAssembly(int tempCountDetails,int generalCountDetail)
                     workerCountDetails = 0;
                     break;
             }
-            {
+            
                 for (int i = 1; i <= workerCountDetails; i++)
                 {
                     
                     tempCountDetails ++;
+                    workerCurrentCountDetails = i;
                     if (tempCountDetails == generalCountDetail) {
-                        Console.WriteLine($"Worker {this.Name} added {workerCountDetails} detail(s) to Agregat");
+                        Console.WriteLine($"Worker {this.Name} added {workerCurrentCountDetails} detail(s) to Agregat");
```

Comment:
you make a lot of useless work with `for` . You can just calculate is **WorkerCapacity (workerCountDetails)** bigger than **left details to assemble** or not